<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitterMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>448</width>
      <height>384</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="0">
       <widget class="QRadioButton" name="rBtnRowSelect">
        <property name="text">
         <string>行选择</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QRadioButton" name="rBtnColSelect">
        <property name="text">
         <string>列选择</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="btnSetHeader">
        <property name="text">
         <string>设置表头</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>设置行数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="btnIniData">
        <property name="text">
         <string>初始化表格数据</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnInsertRow">
        <property name="text">
         <string>插入行</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnAppendRow">
        <property name="text">
         <string>添加行</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="btnDeleteRow">
        <property name="text">
         <string>删除当前行</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnAdRowHgt">
        <property name="text">
         <string>自行调整行高</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnAdColWgh">
        <property name="text">
         <string>自行调整列宽</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>读取表格内容到文本</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="ckbTabEditable">
        <property name="text">
         <string>表格可编辑</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="ckbAlterRowColor">
        <property name="text">
         <string>间隔行底色</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="ckbHeadRowHid">
        <property name="text">
         <string>隐藏行表头</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="ckbHeadColHid">
        <property name="text">
         <string>隐藏列表头</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>单元格选择</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="tableInfo">
      <row>
       <property name="text">
        <string>L1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>L2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>L3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>L4</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>C1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C6</string>
       </property>
      </column>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
